pyproject.toml

```diff
@@ -18,3 +18,22 @@ ignore-regex = "\\\\[fnrstv]"
 markers = [
     "online",
 ]
+
+[tool.ruff]
+select = [
+    "C4",  # flake8-comprehensions
+    "COM", # flake8-commas
+    "E",   # pycodestyle error
+    "F",   # pyflakes
+    "I",   # isort
+    "ISC", # flake8-implicit-str-concat
+    "PIE", # flake8-pie
+    "RUF", # Ruff-specific rules
+    "UP",  # pyupgrade
+    "W",   # pycodestyle warning
+]
+line-length = 110
+target-version = "py38"
+
+[tool.ruff.isort]
```

Comment:
I wonder what to do about the existing block `tool.isort` further up:
```ini
[tool.isort]
line_length = 110
known_first_party = "vcrpy"
multi_line_output = 3
use_parentheses = true
include_trailing_comma = true
```

In particular:
- does ruff read any of that? (assuming: no)
- should it be dropped? (assuming: yes)
- can any of these options ported to ruff more?

Reply:
> does ruff read any of that?

No 😄 

> should it be dropped?

Yes, done!

> can any of these options ported to ruff more?

That's the default ruff configuration, but some settings configurations was not provided yet.